### PR TITLE
Switch ownCloud APT repository, prepare release

### PR DESCRIPTION
**Pull request by drybjed**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -51,16 +51,54 @@ owncloud_edition: 'community'
...
-owncloud_apt_repo_base: 'http://download.opensuse.org/repositories/isv:/ownCloud:'
+owncloud_apt_repo_base: 'https://download.owncloud.org/download/repositories/stable'
 
 
-# .. envvar:: owncloud_apt_repo_key_fingerprint
+# .. envvar:: owncloud_apt_repo_key_id
 #
 # GPG fingerprint for the public key used to sign the APT repository.
-owncloud_apt_repo_key_fingerprint: 'F9EA4996747310AE79474F44977C43A8BA684223'
+owncloud_apt_repo_key_id: '0xAB7C32C35180350A'
```
Comment (ypid) [inline defaults/main.yml]: What about "BCECA90325B072AB1245F739AB7C32C35180350A" instead?

Comment (drybjed) [inline defaults/main.yml]: Not sure about your systems, but on Debian Jessie `gpg` reported wrong key, or couldn't find the key at all in the keyserver, or Ansible returned an error... This should work. I think that ownCloud GPG key is borked somehow, not sure.

Comment (ypid) [inline defaults/main.yml]: Right. The public key seems not to be online on keyservers. But that is ok. The reason for the fingerprint is to pinpoint this particular key.

Tested it and

``` YAML
owncloud_apt_repo_key_id: 'BCECA90325B072AB1245F739AB7C32C35180350A'
```

works. See https://github.com/jchaney/owncloud/pull/12#issuecomment-99544307 for why I think this makes sense even in this case (https).

Comment (drybjed) [inline defaults/main.yml]: OK, I'll check if this can be used when current set of changes is merged.

**Pull request merged**